<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserRadarDialog</class>
 <widget class="QDialog" name="UserRadarDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户雷达</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblStatus">
       <property name="text">
        <string>正在扫描局域网用户...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listUsers"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnAddContact">
       <property name="text">
        <string>添加联系人</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSetRemark">
       <property name="text">
        <string>设置备注</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBlacklist">
       <property name="text">
        <string>黑名单</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnWhitelist">
       <property name="text">
        <string>白名单</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
